iDiff/differs/testDirs/noPackages/not a package/file

```diff
@@ -0,0 +1 @@
+non-empty
```

Comment:
can you rename this file to remove the spaces? probably not an issue but let's not take any risks :)